<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameNthDialogUI</class>
 <widget class="QDialog" name="FrameNthDialogUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame Nth Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Frame Nth Range:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="mStartFrameSpin">
       <property name="minimum">
        <number>-999999999</number>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>to</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="mEndFrameSpin">
       <property name="minimum">
        <number>-999999999</number>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>by</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="mFrameNthSpin">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QRadioButton" name="mRenderFrameNthRadio">
     <property name="text">
      <string>Render Frame Nth</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="mRenderFillFramesRadio">
     <property name="text">
      <string>Render In-Between Frames</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mCopyOptionsGroup">
     <property name="title">
      <string>Render Frame Nth Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QRadioButton" name="mNoCopyRadio">
        <property name="text">
         <string>Dont Copy</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="mCopyToMissingRadio">
        <property name="text">
         <string>Copy To Missing In-Between</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="mCopyToAllRadio">
        <property name="text">
         <string>Copy To All In-Between</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>mCopyToAllRadio</zorder>
     <zorder>mCopyToMissingRadio</zorder>
     <zorder>mNoCopyRadio</zorder>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="ButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>FrameNthDialogUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>FrameNthDialogUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
